httpserver/src/main/kotlin/tbdex/sdk/httpserver/TbdexHttpServer.kt

```diff
@@ -99,7 +99,7 @@ class TbdexHttpServer(private val config: TbdexHttpServerConfig) {
 
       route("/exchanges") {
         post("/{exchangeId}/rfq") {
```

Comment:
Update endpoint as per spec change
https://github.com/TBD54566975/tbdex/pull/222/files#diff-1ab7678648ffe33c3535373210080ab41cd2c38573890fdfcf008700ef14c3c7R279

Reply:
ok. just a side q - would we leave the other endpoints as `/exchanges/{exchangeId}/order|close`?